src/components/Popups/index.tsx

```diff
@@ -30,7 +33,11 @@ const MobilePopupInner = styled.div`
   }
 `
 
-const FixedPopupColumn = styled(AutoColumn)<{ extraPadding: boolean }>`
+const StopOverflowQuery = `@media screen and (min-width: ${MEDIA_WIDTHS.upToMedium + 1}px) and (max-width: ${
+  MEDIA_WIDTHS.upToMedium + 500
```

Comment:
wait... why are we doing this

Reply:
The network selector overflows into the next line - so the popups get cut off. Another solution is just bumping up z-index on popups to cover network selector -maybe thats better?

Reply:
(they only get cut off within that screen size)

Reply:
this smells bad. we're trying to get all the UI elements in just the right spot at each screen size. overlapping feels bad in general, but popups should definitely overlap the network selector. however it might not be z-index but rather the layering, which could make this really time consuming to fix. 

i'm ok with this change for now, but we need to address all our media queries and try to make things more responsive by using the browser's rendering engine instead of media queries and very specific margins

Reply:
Got it - okay lets track that, ill try with z-index for this first in case that works

Reply:
yeah it seems to be layering